Mathlib/Algebra/Category/ModuleCat/Monoidal/Symmetric.lean

```diff
@@ -103,6 +103,12 @@ theorem braiding_inv_apply {M N : ModuleCat.{u} R} (m : M) (n : N) :
 set_option linter.uppercaseLean3 false in
 #align Module.monoidal_category.braiding_inv_apply ModuleCat.MonoidalCategory.braiding_inv_apply
 
+@[simp]
```

Comment:
I'm not sure if this should be a `simp` lemma by default. Because the RHS is quite a long expr.

Reply:
Thank you both for the reviews! I've removed the `@[simp]` on this lemma but added this
```
@[simp]
theorem tensor_μ_apply
    {A B C D : ModuleCat R} (x : A) (y : B) (z : C) (w : D) :
    tensor_μ _ (A, B) (C, D) ((x ⊗ₜ y) ⊗ₜ (z ⊗ₜ w)) = (x ⊗ₜ z) ⊗ₜ (y ⊗ₜ w) := rfl
```
as a `simp` lemma, more in line with other API for the monoidal structure on modules.